e2e/tests/functional/plugins/plot/logPlot.e2e.spec.js

```diff
@@ -181,26 +181,23 @@ async function testLogTicks(page) {
 async function enableEditMode(page) {
     // turn on edit mode
     await page.locator('text=Unnamed Overlay Plot Snapshot >> button').nth(3).click();
```

Comment:
gross locator

Reply:
ooh yeah we should be able to simplify this a LOT. will do that